src/Path.Drag.js

```diff
@@ -18,7 +18,7 @@ L.PathDraggable = L.Draggable.extend({
 
   _onDown: function (e) {
     var first = e.touches ? e.touches[0] : e;
-    this._startPoint = new L.Point(first.clientX, first.clientY);
```

Comment:
In the condition that I moved map first, `this._startPoint` is located at `x:395,y:262`.